proms-resources/frontend/src/main/resources/SLING-INF/content/Extensions/AdminDashboard/DashboardSettingsConfiguration.json

```
@@ -0,0 +1,9 @@
+{
+  "jcr:primaryType": "cards:Extension",
+  "cards:extensionPointId": "cards/coreUI/admindashboard/entry",
+  "cards:extensionName": "Staff dashboard",
```

After playing with the functionality for a while, I think this is a better title:
```suggestion
  "cards:extensionName": "Clinic dashboard",
```